app/models/manageiq/providers/kubernetes/container_manager/refresh_parser.rb

```diff
@@ -362,8 +362,33 @@ def parse_container(container, pod_id)
         :backing_ref     => container.containerID,
         :container_image => parse_container_image(container.image, container.imageID)
       }
-
-      # TODO, state
+      Hash(container.state).each do |key, val|
```

Comment:
Defiantly confusing. Here is what I need to do, I have one of

```
"state": {
  "waiting": {
    "reason": "Image: openshift/origin-deployer:v1.0.4 is ready, container is creating"
  }
}

OR

"state": {
  "running": {
    "StartedAt": somedate
  }
}

OR

"state": {
  "terminated": {
    "exitcode": ...,
    "signal": ...,
    "message": ...,
    ... bunch of other attributes
  }
}
```
One of those (terminated, running, waiting) must be present, some attributes inside state are optional.

what I take from above is that I should patch:

\- Hash(container.state).each do |key, val|
\+ Hash(container.state).first do |key, val|

anything different?